mmv1/templates/terraform/examples/iap_settings.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_compute_backend_service" "default" {
+  name          = "backend-service%{random_suffix}"
```

Comment:
Rather than directly injecting a `random_suffix`, can you inject the name using a `vars` value? That'll render a nice name in the docs, and inject a random value in the test.